docs/page.css

```diff
@@ -159,32 +177,36 @@ strong {
 	font-weight: normal;
 }
 
+
+/* TODO: Duplicate styles in main.css. Needed here cause button is inside the iframe */
```

Comment:
This bit of CSS is duplicated in main.css. Since the floating button anchored at the bottom right corner is a reused component, it makes sense to have the styles in main.css.

However, the button for docs seems to live inside iframe, is it easy to move it outside of iframe and remove the css here? 

@mrdoob